### DISCUSSION TITLE
The copy link function is not available

### DISCUSSION THREAD

**Discussion by jasonDesyn** [WalletConnect/walletconnect-monorepo#3208] [Developer Support]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/89964643/0c6c4fc3-c6e5-45bd-80a4-1ef129c00159)
I use the copy function normally in the local development environment, why is the function unavailable after deploying to the server?

**Comment by jasonDesyn** [comment]
!!!

**Comment by glitch-txs** [comment]
Hi, do you have any public repo/example to reproduce this issue?
Also which framework are you working with?

**Reply by jasonDesyn** [reply]
Here is an open source project reference. You can pull it and run it. The problem I mentioned will appear. There is no problem with local startup. If you leave the local environment to access, an error will occur. You can try LAN ip access.

 https://github.com/balancer/frontend-v2

**Reply by glitch-txs** [reply]
hi, using https://app.balancer.fi/ I'm able to copy
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/66949816/35a713fc-5bd9-4285-8954-ec591408a6e0)

have you tried clearing cache/localStorage?

**Reply by jasonDesyn** [reply]
Cleaning does not work, you are visiting his online project If you run his project locally but not visit it on localhost, you will find the same problem

**Reply by jasonDesyn** [reply]
My project includes other people's projects I just sent to you. I have found the same problem. If you start http://localhost access locally, there is no problem, but as long as you leave the development environment, an error will be reported.![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/89964643/6bd7e485-7403-4048-a4fe-997e139b1938)

**Reply by jasonDesyn** [reply]
I have verified that other people's code will have the same problem, but I don't know why their mainnet will not report an error. Is there any separate configuration? There is no corresponding description in the document

**Reply by jasonDesyn** [reply]
So if the address I visit is https instead of http, I can solve this problem, right?

**Reply by glitch-txs** [reply]
Yes, you can also protect your project ID whitelisting domains in the WalletConnect Cloud dashboard.

**Reply by jasonDesyn** [reply]
Thank you very much for your answer！

**Reply by glitch-txs** [reply]
Hey @jasonDesyn I've just tested this again and works now, you can try generating a new project ID, or cleaning up localStorage
